core-web/libs/sdk/angular/README.md

```diff
@@ -1,229 +1,599 @@
+
+**Usage**:
+
+```typescript
+import { DotCMSBasicContentlet } from '@dotcms/types';
+import { DotCMSBlockEditorRenderer } from '@dotcms/angular/next';
+
+const myCustomRenderers = {
+    customBlock: import('./custom-block.component').then(c => c.CustomBlockComponent)
 };
+
+@Component({
+    selector: 'app-your-component',
+    imports: [DotCMSShowWhen],
+    template: `
+        <dotcms-block-editor-renderer
+            [blocks]="contentlet.myBlockEditorField"
+            [customRenderers]="myCustomRenderers"
+        />
+    `
+})
+export class YourComponent {
+    @Input() contentlet: DotCMSBasicContentlet;
+}
 ```
 
-This way, we will have access to `DOTCMS_CLIENT_TOKEN` from anywhere in our application.
 
-### Client Usage
-To interact with the client and obtain information from, for example, our pages
+**Editor Integration**:
+
+* Renders exactly as configured in dotCMS editor
+* Supports custom renderers for block types
+
+**Common Issues**:
+
+* Should not be used with DotCMSEditableText
+* Inherits styling from parent containers, which may cause conflicts
+* `DotCMSBlockEditorRenderer` only works with Block Editor fields. For other fields, use [`DotCMSEditableText`](#dotcmseditabletext).
+
+📘 For advanced examples, customization options, and best practices, refer to the [DotCMSBlockEditorRenderer README](https://github.com/dotCMS/core/tree/master/core-web/libs/sdk/angular/src/lib/components/DotCMSBlockEditorRenderer).
+
+### Directives
+
+#### DotCMSShowWhen
+
+**Overview**: Directive for conditionally showing content based on the current UVE mode.
+
+**Input**:
+
+| Input    | Type       | Required | Description                                    |
+|----------|------------|----------|------------------------------------------------|
+| `when`   | `UVE_MODE` | ✅        | The UVE mode when content should be displayed  |
+
+**Usage**:
 
 ```typescript
+import { UVE_MODE } from '@dotcms/types';
+import { DotCMSShowWhen } from '@dotcms/angular/next';
+
+@Component({
+    selector: 'app-your-component',
+    imports: [DotCMSShowWhen],
+    template: `
+        <div *dotCMSShowWhen="UVE_MODE.EDIT">
+            Only visible in edit mode
+        </div>
+    `
+})
 export class YourComponent {
-    private readonly client = inject(DOTCMS_CLIENT_TOKEN);
-
-    this.client.page
-        .get({ ...pageParams })
-        .then((response) => {
-            // Use your response
-        })
-        .catch((e) => {
-            const error: PageError = {
-                message: e.message,
-                status: e.status,
-            };
-            // Use the error response
-        })
 }
 ```
 
-For more information on how to use the dotCMS Client, you can visit the [documentation](https://www.github.com/dotCMS/core/blob/main/core-web/libs/sdk/client/README.md)
+**Editor Integration**:
+
+* Useful for mode-based behaviors outside of render logic
+
+**Common Issues**:
+
+* Requires proper mode enum from `@dotcms/types`
```

Comment:
Explain what each entry in the enum means for the UVE state.

Reply:
It's documented in the enum doc!